spec.html

```diff
@@ -4471,20 +4472,20 @@ <h1>Data Blocks</h1>
 
       <emu-clause id="sec-createbytedatablock" aoid="CreateByteDataBlock">
         <h1>CreateByteDataBlock ( _size_ )</h1>
-        <p>The abstract operation CreateByteDataBlock takes argument _size_ (an integer). It performs the following steps when called:</p>
+        <p>The abstract operation CreateByteDataBlock takes argument _size_ (a Number whose mathematical value is an integer). It performs the following steps when called:</p>
```

Comment:
This is an example of a place where I'd use the term "an integral Number".

```suggestion
        <p>The abstract operation CreateByteDataBlock takes argument _size_ (an integral Number). It performs the following steps when called:</p>
```